python/ray/_private/runtime_env.py

```diff
@@ -81,8 +81,9 @@ def __init__(self, runtime_env_json: dict):
         self._dict = dict()
 
         if "working_dir" in runtime_env_json:
-            self._dict["working_dir"] = runtime_env_json["working_dir"]
-            working_dir = Path(self._dict["working_dir"])
+            if "conda" in runtime_env_json or "pip" in runtime_env_json:
+                self._dict["working_dir"] = runtime_env_json["working_dir"]
```

Comment:
should we always put it there?

Reply:
Maybe...  My reasoning here was to not change the codepath for the case when only working_dir is specified and pip/conda are not specified in case it messed something up.  

If we always pass working_dir, then if only working_dir is specified,  it will change the runtime_env_hash to be different from the runtime_env_hash of ordinary workers that don't have a runtime env (such as the idle workers which are pre-started), and this will force new workers to be started.  But if starting these new workers isn't a problem, then I guess we should always pass working_dir.